server/management/commands/import_uc_registrations.py

```diff
@@ -80,6 +80,10 @@ def handle(self, *args: Any, **options: Any) -> None:
                 )
             )
             persons_data = [registration["Person"] for registration in registrations]
+
+            for i in range(len(persons_data)):
+                persons_data[i] = persons_data[i]["email_canonical"].lower()
```

Comment:
shoudnt this be `persons_data[i]["email_canonical"] = persons_data[i]["email_canonical"].lower()` ?

Reply:
oh mb joe, i'll fix this

Reply:
Also, why use range, len instead of just iterating over the elements?

Reply:
@punchagan rookie mistake from me. I forgot that by default objects are always passed by reference (mutable) in python